tests/config-wrapper-malloc-0-null.h

```diff
@@ -0,0 +1,39 @@
+/* config.h wrapper that forces calloc(0) to return NULL.
```

Comment:
In commit message for "Add a test component with malloc(0) returning NULL".

> `Make this a separte job with UBSan (and ASan) to detect potential.`

s/separte/separate

and is there supposed to be a word after "potential" in this sentence?

Reply:
Fixed in rebase.